src/tools/rustfmt/src/comment.rs

```diff
@@ -1,6 +1,6 @@
 // Formatting and tools for comments.
 
-use std::{self, borrow::Cow, iter};
+use std::{borrow::Cow, iter};
```

Comment:
@calebcartwright would we prefer that the rustfmt changes be marked `#[allow(redundant_prelude_imports)]` similar to the ask in https://github.com/rust-lang/rust/pull/118297 for the `unused_tuple_struct_fields` lint?

Reply:
> @calebcartwright would we prefer that the rustfmt changes be marked `#[allow(redundant_prelude_imports)]` similar to the ask in #118297 for the `unused_tuple_struct_fields` lint?

Thank you. Sorry I didn't update the title and description in time. According to petrochenkov's review comments and help, the redundant prelude lint was not added. Instead edit lint unused_imports to check this.